app/javascript/application.js

```diff
@@ -0,0 +1,29 @@
+// Entry point for the build script in your package.json
+// import "@hotwired/turbo-rails"
+import "./controllers"
+import React from 'react';
+import ReactDOM from 'react-dom';
+import {
+  createBrowserRouter,
+  RouterProvider,
+} from "react-router-dom";
+import "./index.css";
+import {Provider} from "react-redux";
+import {store} from "./store.js";	
+
+import Greeting from "./Greeting.js";
+
+const router = createBrowserRouter([
+  {
+    path: "/",
+    element: <Greeting/>,
+  },
+]);
+
+ReactDOM.createRoot(document.getElementById("root")).render(
```

Comment:
- [ ] I can't run the app because there is no react-router-dom, or babel-loader installed, also there are no run scripts so the server can't precompile assets like css. kindly make sure to fix this so your app workes as expected